addons/website/views/website_templates.xml

```diff
@@ -1433,6 +1433,13 @@
     </xpath>
 </template>
 
+<!-- Area for blocks shared by all pages (e.g. all pages popups) -->
+<template id="shared_blocks" inherit_id="website.layout" name="Shared blocks">
+    <xpath expr="//main" position="inside">
+        <div id="o_shared_blocks" class="oe_unremovable"/>
```

Comment:
Is the class actually needed?

Reply:
Mmh... that code is probably broken then. As o_shared_blocks contains the branding, it should naturally be unremovable. But let's ignore that for now, I'll check later. Let's keep the class.